app/enterprise/changelog.md

```diff
@@ -4,6 +4,223 @@ no_search: true
 no_version: true
 ---
 
+## 2.5.0.0 (beta1)
+**Release Date** 2021/07/13
+
+### Features
+
+#### Enterprise
+- Updated banner content informing users about free mode to include the benefits of and a link for updating to Konnect.
+- This release includes several updates to Kong Manager, the Kong Gateway UI, including the following:
+  - Support for the Mocking and Response Rate Limiting plugins.
+  - Updates to the Konnect Plugin page, allowing users to:
+    - Create new plugin associated with `consumer` or `global`.
+    - View plugin details by clicking the row.
+    - Edit and delete plugins that are not associated with a `service` or `route`.
+    - Click the `applied to` entity type (`service`, `route`, or `consumer`) link to view the details of that entity.
+    - Sort the columns.
+    - Search by ID.
+
+#### Core
+- Control Planes can now send updates to new data planes even if the control planes lose connection to the database.
+  [#6938](https://github.com/kong/kong/pull/6938)
+- Kong now automatically adds `cluster_cert`(`cluster_mtls=shared`) or `cluster_ca_cert`(`cluster_mtls=pki`) into
+  `lua_ssl_trusted_certificate` when operating in Hybrid mode. Before, Hybrid mode users needed to configure
+  `lua_ssl_trusted_certificate` manually as a requirement for Lua to verify the Control Plane’s certificate.
+  See [Starting Data Plane Nodes](/gateway-oss/2.5.x/hybrid-mode/#starting-data-plane-nodes)
```

Comment:
For this and subsequent docs links, do we have the documentation in EE as well?  I'd hate to bounce a user from the enterprise changelog over to the OSS docs where they then may have different links

Reply:
At this time there are no Enterprise links bc I do not add Enterprise content until the GA release so as to not confuse users who are not using the beta release. I will be adding this content and it will be available and the links updated for the GA release.